src/components/LinkWrapper/index.tsx

```diff
@@ -31,6 +33,7 @@ const UswdsReactLink = ({
       variant={variant}
       asCustom={RouterLink}
       className={classnames(className)}
+      onClick={onClick}
```

Comment:
I'm a bit torn with using an onClick handler within a component that actually already has one.  I've seen this implementation and it works, but can run into a performance issue with the router.  An alternative solution may be to use a button that handled the onClick, then user history.push to call the route change.   Just a thought/nitpick

Reply:
This shouldn't have to be a problem. It's a matter of access and flexibility. `onClick` is a general property of link elements, the interface for this component just didn't have it included, and I don't think it's overriding any behavior for the router.

The idea was to keep things simple and provide the access on an element that should have had it already, and not have to handle more complicated conditions on whether the target element should be rendered as buttons or navlinks. Since a fair amount of buttons through these flows are tied to route paths, it seemed simplest to leave them as links so that everything works well in their basic form, and then outfit them further as buttons as needed. There are basically no buttons used in the form process as a result.

In terms of performance, I wouldn't consider this area to be critical enough to not use non-standard elements.

Reply:
Sounds good, yep.  Just wanted maybe have a quick discussion, but this makes sense and thanks for the info.